scylladb/sphinx-scylladb-theme
# Makes non latest page clearer

Closes #74

Adds the next warning note if the version being read differs from the latest version published.
The theme automatically detects which is the latest version published reading the environment variable``LATEST_VERSION``. This is set in runtime by the GitHub Action script: https://github.com/scylladb/sphinx-scylladb-theme/blob/master/.github/workflows/pages.yml#L32

![image](https://user-images.githubusercontent.com/9107969/99394094-3d361900-28d6-11eb-8079-ac89e8c7a86e.png)


## How to test this PR

1. Run the command ``make multiversion``.
2. Open the file ``docs/_build/master/examples/index.html`` with your favorite browser. Since LATEST_VERSION is set to master, you should not see any warning.
3. Open the file ``docs/_build/0.1.8/examples/index.html``. You should see the warning.

**Note:** If you want to test locally setting ``LATEST_VERSION`` to another version (e.g. 0.1.13), run in your terminal ``export LATEST_VERSION = 0.1.13`` before ``make multiversion``.
